test/unit/lit/cloud_translation/providers/examples.rb

```diff
@@ -12,14 +12,14 @@ def cloud_provider_examples(described_klass)
 
     describe 'when single string is given' do
       it 'translates single string to target language' do
-        subject.must_match(/\blis\b/)
+        _(subject).must_match(/\blis\b/)
```

Comment:
Why `_(variable)` syntax?

Reply:
minitest as complaining, that old syntax will be deprecated soon